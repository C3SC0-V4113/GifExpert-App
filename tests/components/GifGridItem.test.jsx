import { render } from "@testing-library/react";
import { GifGridItem } from "../../src/components/GifGridItem";

describe("Test in <GifGridItem/>", () => {
  test("debe hacer un match con el snapshot", () => {
    const gif = {
      id: "CFGyQLh90JU7S",
      title: "metal gear solid GIF",
      url: "https://media1.giphy.com/media/CFGyQLh90JU7S/giphy.gif?cid=b3e64bedy9zpam0f6byzw5r8ioi0o3eyugu2h2ryw60q1obm&ep=v1_gifs_search&rid=giphy.gif&ct=g",
    };

    const { container } = render(
      <GifGridItem title={gif.title} url={gif.url} />
    );

    expect(container).toMatchSnapshot();
  });
});
