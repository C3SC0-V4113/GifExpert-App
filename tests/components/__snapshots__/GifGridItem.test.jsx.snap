// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in <GifGridItem/> debe hacer un match con el snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="metal gear solid GIF"
      src="https://media1.giphy.com/media/CFGyQLh90JU7S/giphy.gif?cid=b3e64bedy9zpam0f6byzw5r8ioi0o3eyugu2h2ryw60q1obm&ep=v1_gifs_search&rid=giphy.gif&ct=g"
    />
    <p>
      metal gear solid GIF
    </p>
  </div>
</div>
`;
